engine/graphics/src/null/graphics_null.cpp

```diff
@@ -61,6 +61,26 @@ namespace dmGraphics
 
     DM_REGISTER_GRAPHICS_ADAPTER(GraphicsAdapterNull, &g_null_adapter, NullIsSupported, NullRegisterFunctionTable, g_null_adapter_priority);
 
+    struct AssetContainerLock
+    {
+        AssetContainerLock(dmMutex::HMutex mutex)
+        : m_Mutex(mutex)
+        {
+            if (m_Mutex)
```

Comment:
Here is changes that doesn't match previous implementation. Previously condition looks like
```
if (ctx->m_AssetContainerMutex && ctx->m_UseAsyncTextureLoad)
```
So should it be checked `m_UseAsyncTextureLoad` from context also? See https://github.com/defold/defold/pull/8812/files#diff-42203247c103497473be5308812b1dd9b91b542fa8611fcf5e8d519fbab557afL23-L25